common/interfaces/user_interface/ui_multi_instruction.c

```diff
@@ -151,6 +151,23 @@ static void change_current_index(const lv_key_t PRESSED_KEY)
     }
 }
 
+static void update_text(uint8_t index){
+    lv_label_set_static_text(obj->text, data->instruction_content[index].text);
```

Comment:
check for NULL

Reply:
text is not a pointer but a char array, we could still check if the string is empty but doesn't add any benefit IMO.